modules/post/windows/gather/credentials/filezilla_server.rb

```diff
@@ -249,16 +273,18 @@ def get_filezilla_creds(paths)
     configuration << [config['ftp_port'], config['ftp_bindip'], config['admin_port'], config['admin_bindip'],
       config['admin_pass'], config['ssl'], config['ssl_certfile'], config['ssl_keypass']]
 
+    begin
+      lastser = parse_interface(fsi_xml)
+      lastserver << [lastser['ip'], lastser['port'], lastser['password']]
+      vprint_status("Last Server Information:")
+      vprint_status("         IP: #{lastser['ip']}")
+      vprint_status("       Port: #{lastser['port']}")
+      vprint_status("   Password: #{lastser['password']}")
+      vprint_line
 
-    lastser = parse_interface(fsi_xml)
-    lastserver << [lastser['ip'], lastser['port'], lastser['password']]
-
-    vprint_status("Last Server Information:")
-    vprint_status("         IP: #{lastser['ip']}")
-    vprint_status("       Port: #{lastser['port']}")
-    vprint_status("   Password: #{lastser['password']}")
-    vprint_line
-
+    rescue
```

Comment:
IIRC, we spoke about this on IRC, and the problem was ```doc``` being ```nil``` at https://github.com/nikaiw/metasploit-framework/blob/filezilla/modules/post/windows/gather/credentials/filezilla_server.rb#L446. Is that right?